components/ui/common/subMenu/index.tsx

```
@@ -0,0 +1,51 @@
+import { memo, FC } from "react";
+import dynamic from "next/dynamic";
+import Filters from "@components/ui/filters";
+import useStore from "@store";
+
+interface SubMenuProps {
+  placeLabel: string;
+}
```

ESLint

[error] Do not declare interfaces outside of the /types directory. Centralize all interfaces in /types.

(no-restricted-syntax)